CONTRIBUTING.md

```diff
@@ -21,11 +21,10 @@ Purgecss use [SemVer](http://semver.org/) for versioning.
 
 ### Development Workflow
 
-After cloning Purgecss, run `npm i` to fetch its dependencies. Then, you can run
+After cloning Purgecss, run `npm i && npm run bootstrap` to fetch its dependencies. Then, you can run
 several commands:
 
-* `npm run dev` will build cjs and es module of all PurgeCSS packages in their `lib` folder and
-  watch for changes.
+* `npm run build` will build cjs and es module of all PurgeCSS packages in their `lib` folder.
```

Comment:
Is there a way to activate the watch mode, I'm not familiar with rollup ?

Reply:
There's no watch mode at the moment. The build system was changed when the repository was changed to a monorepo